src/Stratis.Bitcoin/Builder/FullNodeFeatureExecutor.cs

```diff
@@ -54,7 +54,25 @@ public void Initialize()
             try
             {
                 this.Execute(service => service.ValidateDependencies(this.node.Services));
+            }
```

Comment:
I prefer how it was before, I don't see over-reporting as an issue.
or we could have a NodeConfigurationException thrown and the `catch` taking care of these exceptions in a certain way.

Reply:
In any case let's keep this improvement for another PR.

Reply:
Ok. I've reverted the change.